src/TLuaInterpreter.cpp

```diff
@@ -15958,6 +15958,9 @@ void TLuaInterpreter::initLuaGlobals()
     lua_register(pGlobalLua, "getMapRoomExitsColor", TLuaInterpreter::getMapRoomExitsColor);
     lua_register(pGlobalLua, "setMapRoomExitsColor", TLuaInterpreter::setMapRoomExitsColor);
     lua_register(pGlobalLua, "showNotification", TLuaInterpreter::showNotification);
+    lua_register(pGlobalLua, "registerMapInfoProvider", TLuaInterpreter::registerMapInfoProvider);
+    lua_register(pGlobalLua, "killMapInfoProvider", TLuaInterpreter::killMapInfoProvider);
+    lua_register(pGlobalLua, "toggleMapInfoProvider", TLuaInterpreter::toggleMapInfoProvider);
```

Comment:
I don't think `Provider` adds much value here, can be simpler/shorter without it:

```suggestion
    lua_register(pGlobalLua, "registerMapInfo", TLuaInterpreter::registerMapInfoProvider);
    lua_register(pGlobalLua, "killMapInfo", TLuaInterpreter::killMapInfoProvider);
    lua_register(pGlobalLua, "toggleMapInfo", TLuaInterpreter::toggleMapInfoProvider);
```

The idea is to keep it as approachable as possible